forms/tileseteditor.ui

```diff
@@ -271,42 +293,47 @@
                </property>
               </spacer>
              </item>
-             <item row="14" column="0" colspan="3">
-              <widget class="QLineEdit" name="lineEdit_metatileLabel">
-               <property name="clearButtonEnabled">
-                <bool>true</bool>
-               </property>
-              </widget>
-             </item>
-             <item row="1" column="2">
-              <widget class="QComboBox" name="comboBox_layerType"/>
-             </item>
-             <item row="8" column="0" colspan="3">
-              <widget class="NoScrollComboBox" name="comboBox_metatileBehaviors" native="true"/>
```

Comment:
I also want to point out that QtCreator removed `native="true"` from `comboBox_metatileBehaviors`, not sure if that should be restored.

Reply:
none of porymap's other comboboxes have this property so it's probably fine